src/platforms/zephyr/src/libAtomVM/CMakeLists.txt

```diff
@@ -0,0 +1,175 @@
+#
+# This file is part of AtomVM.
+#
+# Copyright 2023 Winford (Uncle Grumpy) <winford@object.stream>
```

Comment:
This file looks based on prior work that has been done on `src/libAtomVM/CMakeLists.txt`, so we should include the original copyright notice too.

Reply:
Right. I forgot to add that yet.

Reply:
I also borrowed heavily from [@adolfogc](https://github.com/adolfogc)'s comments in issue https://github.com/atomvm/AtomVM/issues/802#issuecomment-1711941648, so I feel like we should give Adolfo E. García credit as well. I should probably ask for his explicit permission and ask if he would like his name added as well, as a follow up to his post.